Middleware: Add CSP Report Only support (#58074)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

### content_security_policy_template

> Set the policy template that will be used when adding the Content-Security-Policy header to your requests. `$NONCE` in the template includes a random nonce.

Reviewer: ```suggestion
Set the policy template that will be used when adding the `Content-Security-Policy` header to your requests. `$NONCE` in the template includes a random nonce.
```

### content_security_policy_report_only

> Set to `true` to add the Content-Security-Policy-Report-Only header to your requests. CSP in Report Only mode enables you to experiment with policies by monitoring their effects without enforcing them.

Reviewer: ```suggestion
Set to `true` to add the `Content-Security-Policy-Report-Only` header to your requests. CSP in Report Only mode enables you to experiment with policies by monitoring their effects without enforcing them.
```

You can enable both policies simultaneously.

### content_security_policy_template

> Set the policy template that will be used when adding the Content-Security-Policy-Report-Only header to your requests. `$NONCE` in the template includes a random nonce.

Reviewer: ```suggestion
Set the policy template that will be used when adding the `Content-Security-Policy-Report-Only` header to your requests. `$NONCE` in the template includes a random nonce.
```

Set Content Security Policy template used when adding the Content-Security-Policy header to your requests. `$NONCE` in the template includes a random nonce.

### content_security_policy_report_only

> Set to `true` to add the Content-Security-Policy-Report-Only header to your requests. CSP in Report Only mode allows to experiment with policies by monitoring their effects without enforcing.

Reviewer 2: ```suggestion
Set to `true` to add the Content-Security-Policy-Report-Only header to your requests. CSP in Report Only mode enables you to experiment with policies by monitoring their effects without enforcing them.
```

> Both policies can be enabled simultaneously.

Reviewer 2: ```suggestion
You can enable both policies simultaneously.
```

### content_security_policy_template

> Set Content Security Policy Report Only template used when adding the Content-Security-Policy-Report-Only header to your requests. `$NONCE` in the template includes a random nonce.

Reviewer 2: I'm not sure what you are trying to say here. Is it: "Set the Content Security Policy Report Only template that is used when you add the Content-Security-Policy-Report-Only header to your requests."?

Author: @chri2547 I updated the wording of this phrase, hopefully it makes more sense now.

Reviewer 2: 👍
